### ISSUE TITLE
自定义端口, 重启会导致http.json被错误的重复替换

### ISSUE THREAD

**Issue by HimekoEx** [AirryCo/spt-server-ci#4]
原本`6969`
我调整成`46969`
结果重启镜像会变成`446969`, 再重启类推

**Comment by HimekoEx** [comment]
![image](https://github.com/user-attachments/assets/932be0a0-af41-4052-97d1-2f2973ebaee4)

**Comment by HimekoEx** [comment]
docker compose file

```yaml
services:
  spt-server:
    image: stblog/spt-server:3.10.0
    container_name: spt-server
    restart: unless-stopped
    volumes:
      - /root/spt_server_data:/opt/spt-server
    ports:
      - '46969:46969'
    environment:
      - TZ=Asia/Shanghai
      - backendIp=xxx.xxx.xxx.xxx
      - backendPort=46969
```

**Comment by AirryCo** [comment]
把container down掉，重新在compose.yaml里定义端口，看看还会不会出现这种情况。

**Comment by HimekoEx** [comment]
会的, 因为做了`volume`持久化, 每次重启容器, 都会错误的被替换掉
`docker/entrypoint.sh` 的 `sed -i "s/6969/${PORT}/g" SPT_Data/Server/configs/http.json` 可以优化一下

**Comment by HimekoEx** [comment]
一旦出现错误的替换, 我只能在容器启动前, 将现有的`http.json`自定义的 prot 改回 `6969` 才能正常启动

**Comment by AirryCo** [comment]
该问题已在3.10.1镜像中解决。当然，你也可以通过端口映射来规避此问题。

The issue has been resolved in the 3.10.1 image. Of course, you can also work around this issue by using port mapping.

**Comment by AirryCo** [comment]
该问题已在3.10.1镜像中解决。当然，你也可以通过端口映射来规避此问题。

The issue has been resolved in the 3.10.1 image. Of course, you can also work around this issue by using port mapping.
